### ISSUE TITLE
Working code

### ISSUE THREAD

**Issue by apauliuc** [MrRobot2211/deep-photo-enhance-t2#1]
Hi. I greatly appreciate your work on upgrading the DPE model to TF2, and thanks for your effort. I'm wondering, have you managed to run the prediction code using any provided model? I cloned your repo but get big errors when trying to run it. Thanks!

**Comment by MrRobot2211** [comment]
Yes i managed to run it using python predict.py it just uses the pretrained
736

**Comment by vdivakar** [comment]
Hi, how to run the training code? I want to train the model on my custom dataset.

**Comment by MrRobot2211** [comment]
Hi , you should run train.py and set the images in the directories as explained in the original dpe repo as well as setup the corresponding files . That is you need to have an input folder that conttains LPGAn and inside a folder called input with the original pictures and another one called labels with target pictures. Inside LPGAn you should add files train_input.txt, test_input.txt and test.txt (in there you should have the images that are on both folders , that's to say tthe ones you have the original and the translation) . THis works forr images that are of maximum size 512 in any direction, you can use the resize.py script for that. I am adding a clear readme for the whole prrocess today.

**Comment by titsitits** [comment]
> Hi , you should run train.py and set the images in the directories as explained in the original dpe repo as well as setup the corresponding files . That is you need to have an input folder that conttains LPGAn and inside a folder called input with the original pictures and another one called labels with target pictures. Inside LPGAn you should add files train_input.txt, test_input.txt and test.txt (in there you should have the images that are on both folders , that's to say tthe ones you have the original and the translation) . THis works forr images that are of maximum size 512 in any direction, you can use the resize.py script for that. I am adding a clear readme for the whole prrocess today.

Hi,
I would also be interested to try your model in inference (on the pretrained model). Any news about a readme ? ;)
